Makefile

```diff
@@ -16,12 +16,12 @@ build-docker-image:
 
 .PHONY: push-docker-image
 push-docker-image: #build-docker-image
-	docker pull dockercloud/haproxy:$(HAPROXY_VER)
-	docker pull tutum/redis
-	docker tag dockercloud/haproxy:$(HAPROXY_VER) taskcluster/haproxy:$(HAPROXY_VER)
-	docker push taskcluster/haproxy:$(HAPROXY_VER)
-	docker tag tutum/redis:latest taskcluster/redis:latest
-	docker push taskcluster/redis:latest
+	# docker pull dockercloud/haproxy:$(HAPROXY_VER)
```

Comment:
I'd like to move back to docker cloud for the whole deployment at some point, but you're write, I should delete them and just refer to history!